Linux/Sources/ContextAgent.cpp

```diff
@@ -13,15 +13,15 @@
 
 bool _xerror = false;
 
-int ContextAgent::x11Errorhandler(Display *display, XErrorEvent *error) {
+int ContextAgent::x11Errorhandler(__attribute__((__unused__)) Display *display, __attribute__((__unused__)) XErrorEvent *error) {
```

Comment:
Don't use compiler attribute. We aim for portable code. It's ok to add a flag to not raise a warning for those.